airbyte-server/src/main/java/io/airbyte/server/handlers/SchedulerHandler.java

```diff
@@ -234,18 +241,18 @@ public JobStatusRead syncConnection(final ConnectionIdRequestBody connectionIdRe
         .build());
 
     return new JobStatusRead()
-        .status(job.getStatus().equals(JobStatus.COMPLETED) ? io.airbyte.api.model.JobStatus.SUCCEEDED : io.airbyte.api.model.JobStatus.FAILED);
+        .status(job.getStatus().equals(JobStatus.COMPLETED) ? JobStatusReadStatus.SUCCEEDED : JobStatusReadStatus.FAILED);
```

Comment:
I honestly prefer the conflict than this name. By suffixing it with "ReadStatus" you prevent using this enum in other api models.

Reply:
okay. i will try to address this in the API PR.

Reply:
basically i will see if i can get rid of this completely and just replace it with the existing JobRead struct. not really a good reason we have these separate structs right now.